js/admin/notifications.js

```diff
@@ -22,151 +22,120 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-$(document).ready(function() {
-	var hints = $('.translatable span.hint');
-	if (youEditFieldFor)
-		hints.html(hints.html() + '<br /><span class="red">' + youEditFieldFor + '</span>');
-
-	var html = "";
-	var nb_notifs = 0;
-	var wrapper_id = "";
-	var type = new Array();
+$(document).ready(function () {
+  if (youEditFieldFor) {
+    $('.translatable span.hint').append('<br /><span class="red">' + youEditFieldFor + '</span>')
+  }
 
-  $('.notification.dropdown-toggle').on('click', function (event) {
+  $('.notification.dropdown-toggle').on('click', function () {
     $(this).parent().toggleClass('open');
     updateEmployeeNotifications();
   });
 
-  $('body').on('click', function (e) {
-    if (!$('#notification.dropdown').is(e.target)
-      && $('#notification.dropdown').has(e.target).length === 0
-      && $('.open').has(e.target).length === 0
-    ) {
-      if ($('#notification.dropdown').hasClass('open')) {
-        getPush();
-      }
-      $('#notification.dropdown').removeClass('open');
+  $(document).on('click', function (e) {
+    if (!$(e.target).closest('#notification').length && $('#notification').hasClass('open')) {
+      $('#notification').removeClass('open');
+      getPush();
     }
   });
 
   $('.notifications .nav-link').on('shown.bs.tab', function () {
     updateEmployeeNotifications();
   });
 
-	// call it once immediately, then use setTimeout
+  // call it once immediately, then use setTimeout
   if (parseInt(show_new_orders) || parseInt(show_new_customers) || parseInt(show_new_messages)) {
-	  getPush();
+    getPush();
   }
-
 });
 
 function updateEmployeeNotifications() {
   $.post(
     admin_notification_push_link,
     {
-      "type": $('.notifications .nav-item.active a').attr('data-type')
+      type: $('.notifications .nav-item.active a').data('type')
     }
   );
 }
 
-function getPush()
-{
-	$.ajax({
-		type: 'POST',
-		headers: {"cache-control": "no-cache"},
-		url: admin_notification_get_link+'&rand=' + new Date().getTime(),
-		async: true,
-		cache: false,
-		dataType : 'json',
-		success: function(json) {
-			if (json)
-			{
-				// Set moment language
-				moment.lang(full_language_code);
+function renderOrderNotification(value) {
+  var html = "<a class='notif' href='" + baseAdminDir + "index.php?tab=AdminOrders&token=" + token_admin_orders;
+  html += "&vieworder&id_order=" + value.id_order + "'>";
+  html += "#" + value.id_order + " - ";
+  html += from_msg + "&nbsp;<strong>" + value.customer_name + "</strong>";
+  html += " (" + value.iso_code + ")";
+  html += "<strong class='pull-right'>" + value.total_paid + "</strong>";
+  if (value.carrier !== "") {
+    html += " - " + value.carrier;
+  }
+  return html + "</a>";
```

Comment:
```suggestion
  return `<a class='notif' href='${baseAdminDir}index.php?tab=AdminOrders&token=${token_admin_orders}`
      + `&vieworder&id_order=${value.id_order}'>`
      + `#${value.id_order} - ${from_msg}"&nbsp;`
      + `<strong>${value.customer_name}</strong>`
      + ` (${value.iso_code})`
      + `<strong class='pull-right'>${value.total_paid}</strong>`;
      + (value.carrier !== "" ? ` - ${value.carrier}` : "")
      + "</a>";
```

Reply:
Could you apply this templating in all above functions ?

Reply:
If I could use template strings, could I remove the +?